### ISSUE TITLE
Failed to compile any new css file like the one from Datepicker

### ISSUE THREAD

**Issue by giannis-dallas** [ahmadawais/Gutenberg-Boilerplate#132]
## Bug Report

Hello, I am trying to add react-datepicker to my plugin. Unfortunately, the CSS is not compiling.
I tried to import another CSS file but still, it's not compiling.

Update: I updated, node and npm, I created a new guten-block and again the new `test.scc` file cannot compile

does the config part below means that only the 2 existing files available for compiling? Do I need to eject the project in order to add more files?
```
,
			{
				test: /style\.s?css$/,
				exclude: /(node_modules|bower_components)/,
				use: blocksCSSPlugin.extract( extractConfig ),
			},
			{
				test: /editor\.s?css$/,
				exclude: /(node_modules|bower_components)/,
				use: editBlocksCSSPlugin.extract( extractConfig ),
			},
```

### Expected Behavior

`react-datepicker/dist/react-datepicker.css` is imported as expected to the plugin
`import './test.scss';` to be working

### Actual Behavior

Failed to compile
```
/node_modules/react-datepicker/dist/react-datepicker.css Module parse failed: Unexpected token (1:0)
You may need an appropriate loader to handle this file type.
```

### Steps to Reproduce

Created a new  js file to add my calculator starting with

```
import React from "react";
import DatePicker from "react-datepicker";
 
import "react-datepicker/dist/react-datepicker.css";
```

After compile failed I tried to include file from `\src\block\block.js`
I even tried moving the file to the src folder but nothing worked so far.

For testing purposes, I create `test.scss` file inside the block folder
```
.test{
    background: red;
}
```

this gives again
```
Module parse failed: Unexpected token (1:0)
You may need an appropriate loader to handle this file type.
| .test{
|     background: red;
| }
```

1. `node -v`:  v8.11.1
2. `npm -v`: 6.4.1
3. `yarn --version` (if you use Yarn):
4. `npm ls cgb-scripts` (if you haven’t ejected): -- cgb-scripts@1.14.0

Then, specify:

1. Operating system: Windows and VS Code
2. Browser and version (if relevant): chrome

**Comment by muhammadawais93** [comment]
Getting the exactly same issue.

**Comment by skorasaurus** [comment]
> does the config part below means that only the 2 existing files available for compiling? Do I need to eject the project in order to add more files?

Yes, the webpack config file (generated by create-guten-block and located at https://github.com/ahmadawais/create-guten-block/blob/a166a7f085839114a7c14a305eca743d91b8de6e/packages/cgb-scripts/config/webpack.config.prod.js#L68
means that only 3 files are available for compiling: 
editor.scss, style.scss and also common.scss (a few lines earlier up). 

In order to add more files, you will need to eject the plugin, and [then you can edit the webpack config files which will now be in your ejected block](https://github.com/ahmadawais/create-guten-block/tree/a166a7f085839114a7c14a305eca743d91b8de6e/examples/04-multi-block-ejected/config) to decide which external files (JS or SCSS, or whatever) you want to include. 
You can some basics about webpack at https://webpack.js.org/concepts/

(I'm not 100% sure on understanding this though)
